### ISSUE TITLE
A11y: Elements have missing `checked`/`expanded` attributes

### ISSUE THREAD

**Issue by angelathe** [CDCgov/dibbs-ecr-viewer#1147]
## What needs to be done

The following elements need an html `checked` or `expanded` attribute. Please check first to confirm that they don't have the `checked attribute (manual scans were a bit buggy).

Areas that need `checked` attribute updates:
- Create User form - Selecting program area checkboxes
- Filter by Date in the Library - each date input
- Filter by Reportable Condition - each reportable condition checkbox

Areas that need `expanded` attribute updates:
- Dropdown button in eCR Viewer for multiple versions of an eCR

Example (Create User):
<img width="1864" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMUpuQ3c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--61c3e9013631280f9118459414ad6608f6c14d98/Screenshot%202025-09-16%20at%2009.43.36.png" alt="Screenshot 2025-09-16 at 09.43.36.png" />

## Why it needs to be done

This follows the accessibiility audit scans of the eCR Viewer pages.

## Acceptance Criteria

- [ ] The necessary `checked` attributes are added to the input elements that need them
- [ ] The necessary `expanded` attributes are added to the input elements that need them
- [ ] Manual scans have been run again on these pages/elements, and have come up with no issues.

## Technical Notes

Note: Running the manual guided scans will probably require a pro license. Manual scans for this ticket may specifically require running the `Interactive Elements` test for each area.

- Ex. Filter by Date (eCR Library): [Axe Issue](https://axe.deque.com/issues/af5799b9-0f92-468f-a783-7783d65031d0)
- Ex. Filter by Reportable Condition (eCR Library): [Axe Issue](https://axe.deque.com/issues/7043c07c-9709-40f2-aa9d-768eef10921f)
- Ex. Select program in Create user form: [Axe Issue](https://axe.deque.com/issues/a492c7c6-71b7-4463-abe0-72b8b8c2522b)
- Ex. Multiple eCR versions in the library: [Axe Issue](https://cdc-devtools.dequecloud.com/issues/cb3e68ce-12f4-4c7c-8682-83063f1f69b6)

**Comment by austin-hall-skylight** [comment]
Note about existing behavior: radio button for "Last 24 hours" in date filter has checked property but keeps the checked property even when other date filters are selected. Those newly selected radio buttons don't gain the checked property.
